be/src/common/config.h

```diff
@@ -759,6 +758,10 @@ CONF_String(starlet_cache_dir, "");
 
 CONF_Int64(lake_metadata_cache_limit, /*2GB=*/"2147483648");
 
-} // namespace config
+CONF_mBool(enable_librdkafka_debug_mode, "false");
```

Comment:
how about `dependency.librdkafka.debug_mode.enable`?
and we name it "dependency_librdkafka_debug_mode_enable"

Reply:
Fixed